app/views/podcasts/show.rss.builder

```diff
@@ -155,7 +155,7 @@ xml.rss "xmlns:atom" => "http://www.w3.org/2005/Atom",
               url: ep.enclosure_url(@feed))
           end
 
-          xml.content(:encoded) { xml.cdata!(ep.content) } unless ep.content.blank?
+          xml.content(:encoded) { xml.cdata!(episode_description(ep)) }
```

Comment:
So now `<content:encoded>` will be identical to `<description>` - do we really need both?

Reply:
I see multiple rss hosts that do provide both